docs/source-fabric/fundamentals/precision.rst

```diff
@@ -112,7 +121,50 @@ It is also possible to use BFloat16 mixed precision on the CPU, relying on MKLDN
 .. note::
 
     BFloat16 may not provide significant speedups or memory improvements, offering better numerical stability.
-    For GPUs, the most significant benefits require `Ampere <https://en.wikipedia.org/wiki/Ampere_(microarchitecture)>`_ based GPUs, such as A100s or 3090s.
+    For GPUs, the most significant benefits require `Ampere <https://en.wikipedia.org/wiki/Ampere_(microarchitecture)>`_ based GPUs or posterior, such as A100s or 3090s.
+
+
+----
+
+
+*****************************************************
+Float8 Mixed Precision via Nvidia's TransformerEngine
+*****************************************************
+
+`Transformer Engine <https://github.com/NVIDIA/TransformerEngine>`__ (TE) is a library for accelerating models on the
+latest NVIDIA GPUs using 8-bit floating point (FP8) precision on Hopper GPUs, to provide better performance with lower
+memory utilization in both training and inference. It offers improved performance over true half precision with no degradation in accuracy.
+
+Using TE requires replacing some of the layers in your model. Fabric automatically replaces the :class:`torch.nn.Linear`
+and :class:`torch.nn.LayerNorm` layers in your model with their TE alternatives, however, TE also offers
+`fused layers <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/pytorch.html>`__
+to squeeze out all the possible performance. If Fabric detects that any layer has been replaced already, automatic
+replacement is not done.
+
+This plugin is a mix of "mixed" and "true" precision. The computation is downcasted to FP8 precision on the fly, but
+the model and inputs can be kept in true full or half precision.
```

Comment:
I'm not sure why you wrote 

> This plugin is a mix of "mixed" and "true" precision.

Maybe because not all weights are the same precision? This is generally going to be the case for these precision regimes so this can just be referred to mixed precision training in general. I wouldn't call it a mix of mixed and true.

Reply:
What I mean is that it can convert the weights and data etc into half precision, (like 16-true), but also downcast activations into 8 bit precision (like 16-mixed)